examples/02_decoding/plot_haxby_grid_search.py

```diff
@@ -46,12 +47,12 @@
 print('Functional nifti image (4D) are located at: %s' % haxby_dataset.func[0])
 
 # Load the behavioral data
-labels = np.recfromcsv(haxby_dataset.session_target[0], delimiter=" ")
+labels = pd.read_csv(haxby_dataset.session_target[0], delimiter=" ")
```

Comment:
Should it be "sep" rather than "delimiter"?

Reply:
Patched this for consistency, thanks!